app/scripts/constants.js

```diff
@@ -189,6 +189,7 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     BuildConfigInstantiateFailed: true,
     // Deployment
     DeploymentCancelled: true,
+    RolloutCancelled: true,
```

Comment:
nit: better to keep it alphabetical by type so it's easier to find things

Reply:
If this is for deployment configs, better to move it under that section

Reply:
will do.